src/tty-ask-password-agent/tty-ask-password-agent.c

```diff
@@ -210,6 +229,69 @@ static int ask_password_plymouth(
         return 0;
 }
 
+static void free_consoles(struct console *con, const unsigned int num) {
+        unsigned int n;
+        if (!con || !num)
+                return;
+        for (n = 0; n < num; n++)
+                free(con[n].tty);
+        free(con);
+}
+
+static const char *current_dev = "/dev/console";
+static struct console* collect_consoles(unsigned int * num) {
```

Comment:
The coding style should be in line with the rest of our code base. In particular, it should be `char *foo`, not `char* foo` or `char * foo`.